src/Symfony/Component/Intl/Transliterator/EmojiTransliterator.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Intl\Transliterator;
+
+class EmojiTransliterator
+{
+    private static \Transliterator $instance;
+
+    public static function getInstance(): \Transliterator
+    {
+        if (!isset(static::$instance)) {
```

Comment:
using `static` to access a private static property is a bug. If you extend that class, it will try to access `ChildClass::$instance`, which is not guaranteed to exists. Use `self::$instance` (and probably make the class final as it is not designed for extensibility)

Reply:
Thanks, Fixed.